src/main/resources/application-dev.yml

```diff
@@ -0,0 +1,22 @@
+#
+# The "-dev" profile is for running the API service locally against pre-existing
+# docker containers that are listening on known port numbers for hmpps-auth (8090) and
+# a postgres database (5432).
+#
+
```

Comment:
Configuring the resource server properties - running locally, this services checks that tokens are signed by a locally-held public key. This matches what a locally-running `hmpps-auth` service uses for its `dev` profile.